test/src/test.c

```diff
@@ -9,10 +10,12 @@
 #include "pos.h"
 #include "tiny.h"
 
+#ifdef TINY_COMPILER
```

Comment:
Since the common case is to compile with the compiler included, I think it makes more sense to have this be an `#ifndef TINY_VM_ONLY` or something to that effect

Reply:
Could be. But the code that chooses to add is the compiler, not the VM. In fact the VM always exists in compilation. That's why it seems more accurate to me to refer to the compiler and not to the VM